crates/bevy_ui/src/widget/image.rs

```diff
@@ -73,20 +70,20 @@ impl Measure for ImageMeasure {
     }
 }
 
+/// The filter type for `update_image_content_size_system` and `update_atlas_content_size_system`.
+#[cfg(feature = "bevy_text")]
+pub type UpdateImageFilter = (With<Node>, Without<bevy_text::Text>);
+/// The filter type for `update_image_content_size_system` and `update_atlas_content_size_system`.
+#[cfg(not(feature = "bevy_text"))]
+pub type UpdateImageFilter = With<Node>;
```

Comment:
This seems like a nice refactor. It's annoying the type aliases need to public though, maybe we could make the systems private and add an `ImageSizeSystem` system set?